tss-esapi/tests/context_tests/tpm_commands/symmetric_primitives_tests.rs

```diff
@@ -61,9 +57,11 @@ mod test_hmac {
         let key_pub = Tpm2BPublicBuilder::new()
             .with_type(TPM2_ALG_KEYEDHASH)
             .with_name_alg(TPM2_ALG_SHA256)
-            .with_parms(PublicParmsUnion::KeyedHashDetail(KeyedHashParms::HMAC {
-                hash_alg: HashingAlgorithm::Sha256,
-            }))
+            .with_parms(PublicParmsUnion::KeyedHashDetail(KeyedHashParameters::new(
+                KeyedHashScheme::Hmac {
+                    hmac_scheme: HmacScheme::new(HashingAlgorithm::Sha256),
+                },
```

Comment:
Do we use this `KeyedHashScheme` a lot, or do we expect it to be used a lot? I can see a `const` implemented on it (like `AES_256_CFB` for ciphers), something like `HMAC_SHA256`

Reply:
Not really sure but does not hurt to add it.